kv/db.go

```diff
@@ -163,57 +163,57 @@ func (s *rpcDBServer) executeCmd(args proto.Request, reply proto.Response) error
 	return nil
 }
 
-// Get .
```

Comment:
this part of the change is a bit nonsensical, but
![](http://cdn.meme.am/instances/52000344.jpg)